tasks/main.yml

```diff
@@ -34,8 +34,14 @@
     group: "{{ az_devops_agent_user }}"
     creates: "{{ az_devops_agent_folder }}/config.sh"
 
+- name: Install Dependencies via shipped script
+  command: ./bin/installdependencies.sh
```

Comment:
Does this remove the need for the first task to install requirements (first task)?

Reply:
Also, your not checking if it's changed or not.  It isn't idempotent.

Reply:
- It removes the need for `libcurl` but not for `libunwind8`. -> curl removed
- retry(2) and register result for task